block/submit_loop_test.go

```diff
@@ -68,10 +54,7 @@ func testSubmitLoopInner(
 					return
 				default:
 				}
-				// producer shall not get too far ahead
-				absoluteMax := (args.batchSkew + 1) * args.batchBytes // +1 is because the producer is always blocked after the fact
-				nProduced := nProducedBytes.Load()
-				require.True(t, nProduced < absoluteMax, "produced bytes not less than maximum", "nProduced", nProduced, "max", absoluteMax)
```

Comment:
what is being tested if this is removed?

Reply:
fixed